python/ccxt/base/exchange.py

```diff
@@ -1019,8 +1019,10 @@ def extract_params(string):
     def implode_params(string, params):
         if isinstance(params, dict):
             for key in params:
-                if not isinstance(params[key], list):
-                    string = string.replace('{' + key + '}', str(params[key]))
+                value = params[key]
```

Comment:
Can you please give an example that would crash this code? (just to understand it better)

Reply:
this example link was reference in toppost: https://github.com/ccxt/ccxt/issues/12719